darkmoon/api/v1/metadata/schema.py

```diff
@@ -1,8 +1,8 @@
-"""Imports the modules/classes Field, BaseModel, and Optional."""
-
 from beanie import PydanticObjectId
 from pydantic import BaseModel, Field
 
+"""Import module/class PydanticObjectID, BaseModel, and Field"""
```

Comment:
This did not need to be moved. In fact, the docstring itself at the top should more so explain the purpose of the file itself. Imports don't need docstrings